force-app/main/default/lwc/permanentIdCard/__tests__/permanentIdCard.test.js

```diff
@@ -0,0 +1,98 @@
+import { createElement } from "lwc";
+import PermanentIdCard from "c/permanentIdCard";
+
+describe("c-permanent-id-card", () => {
+  afterEach(() => {
+    // The jsdom instance is shared across test cases in a single file so reset the DOM
+    while (document.body.firstChild) {
+      document.body.removeChild(document.body.firstChild);
+    }
+  });
+
+  it("test buttons", () => {
+    // ARRANGE
+    const element = createElement("c-permanent-id-card", {
+      is: PermanentIdCard
+    });
+
+    element.cardData = {
+      MemberId: "MemberId"
+    };
+
+    element.permData = {
+      PNGBack: "PNGBack",
+      PNGFront: "PNGFront"
+    };
+
+    window.navigator = {
+      userAgent: "CommunityHybridContainer"
+    };
+
+    element.dispatchEvent = jest.fn();
+
+    // ACT
+    document.body.appendChild(element);
+
+    // ASSERT
+    const targetId = element.shadowRoot.querySelector('div[class="id-card permanent"]').id;
```

Comment:
in salesforce never check against the id property in html elements, when being executed salesforce can change that id to anything instead check the data-target-id